Add Calculated Fields section to historical_jobs.md (#27850)
DataDog/documentation: content/en/security/cloud_siem/historical_jobs.md

    {{< img src="security/security_monitoring/detection_rules/convert-historical-job-result-to-signal.png" alt="The results panel for a Historical Job" style="width:90%;" >}}
1. Click **Convert to Signals**.

> ## Calculated Fields Queries

Reviewer: ```suggestion
## Calculated fields queries
```

You can create [Calculated Fields][4] directly within a Historical Job query to define a computed field from existing data sources.

Use calculated fields to transform and enrich your query with formulas to:

> 1. Manipulate text
> 1. Perform arithmetic
> 1. Evaluate conditional logic

Reviewer: Recommend an unordered list unless these must be completed in order.

```suggestion
- Manipulate text
- Perform arithmetic
- Evaluate conditional logic
```

> Once defined, a calculated field can be used like any log attribute for analysis of the job, search across job results, and even defining other calculated fields.

Reviewer: ```suggestion
A calculated field can be used like any log attribute for job analysis, searching job results, and defining other calculated fields.
```

> From the Add menu:

Reviewer: ```suggestion
1. Navigate to the [Historial Jobs list][3].
1. Click **New Job**.
```

> 1. Navigate to [Define Search Queries][6].

Reviewer: ```suggestion
1. Find the [Define Search Queries][6] section.
```

> 1. Click the Add button next to the search bar.
> 1. Select Calculated field.

Reviewer: ```suggestion
1. Click **Add**.
1. Select **Calculated Fields** from the options.
```

See [Calculated Fields Expression Language][5] for the available functions and operators.

## Further Reading

{{< partial name="whats-next/whats-next.html" >}}

[1]: https://app.datadoghq.com/security/rules
[2]: https://app.datadoghq.com/security/configuration/siem/rules/new-job?product=siem
[3]: https://app.datadoghq.com/security/detections/historical-jobs

> [4]: https://docs.datadoghq.com/logs/explorer/calculated_fields/
> [5]: https://docs.datadoghq.com/logs/explorer/calculated_fields/expression_language/

Reviewer: ```suggestion
[4]: /logs/explorer/calculated_fields/
[5]: /logs/explorer/calculated_fields/expression_language/
```
